doc/repro.rst

```diff
@@ -313,6 +313,12 @@ postfix
 
 Some external model names include:
 
+MESSAGE V.3
+   .. todo:: Expand with a list of cases in which this model name has been used.
```

Comment:
Given the PR title/description, it sounds like the AR6 database is a case in which this model name has been used. If so, why not record that here?